api/db-cdr/changelog/db.changelog-12.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+
+  <changeSet author="biopete" id="changelog-12.0">
+    <validCheckSum>ANY</validCheckSum>
+    <preConditions onFail="MARK_RAN">
+      <tableExists tableName="concept" schemaName="cdr"></tableExists>
+      <tableExists tableName="ACHILLES_results" schemaName="cdr"></tableExists>
+    </preConditions>
+    <createView replaceIfExists="true" schemaName="cdr" viewName="ACHILLES_results_view">
+      SELECT
+      `a`.`analysis_id` AS `analysis_id`,
```

Comment:
Is there a reason all the table and column names in here have backquotes around them?

Where did this SQL come from?

Reply:
I copied the  create view statement for the view from my mysql client , Sequel Pro. Backtics don't hurt. They are required if you're table name is a reserved word so mysql uses them a lot.